src/content/1.0.0/docs/start/anywhere.md

```diff
@@ -96,12 +96,13 @@ Property | Options | Description
 `fdb-group-private-key-file` | `file path` | If fdb-group-private-key is not provided, we'll look for it in this file. If not found in this file, we'll generate a default one and place it in this file.
 `fdb-group-servers` | `server-id@host:port, server-id@host:port` | List all servers participating in ledger-group with format of server-id@host:port. All tx-group servers should have this same config.
 `fdb-group-this-server` | `server-id` | Specify which of the above listed server-ids is this server. Note this must be unique for every server in the tx-group, and is likely easiest to supply this setting via environment variable.
+`fdb-group-port`| `int` | Optional. Only needed, if fdb-this-server is not set or or does not point to one of the group-servers. External port to listen on for ledger communication i.e. fdb-group-port=9795.
```

Comment:
- I think the comma after "Only needed" isn't needed.
- Two or's in "is not set or or"
- "e.g." is probably more appropriate here than "i.e." since it's an example